tests/functional/verification_experiments.js

```diff
@@ -105,13 +105,18 @@ define([
       return this.remote
         .setFindTimeout(intern.config.pageLoadTimeout)
         .get(require.toUrl(EXP_SHOWPASSWORD_URL + EXP_CONTROL))
+        .findByCssSelector('.password')
```

Comment:
Use the [type helper function](https://github.com/mozilla/fxa-content-server/blob/master/tests/functional/lib/helpers.js#L1281) instead of low level access here.